### PR TITLE
Add some logs to CLA plugin.

### PR DESCRIPTION
**Pull request by spxtr**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1147)

### CODE REVIEW

File: prow/plugins/cla/cla.go [hunk 1]
```diff
@@ -88,12 +88,14 @@ func handle(gc gitHubClient, log *logrus.Entry, se github.StatusEvent) error {
 
 	org := se.Repo.Owner.Login
 	repo := se.Repo.Name
+	log.Info("Searching for PRs matching the commit.")
```
Comment (foxish) [inline prow/plugins/cla/cla.go]: Is this log useful? Can we get some more information in here, like org, repo?

File: prow/plugins/cla/cla.go [hunk 2]
```diff
@@ -88,12 +88,14 @@ func handle(gc gitHubClient, log *logrus.Entry, se github.StatusEvent) error {
...
 	issues, err := gc.FindIssues(url.QueryEscape(fmt.Sprintf("%s repo:%s/%s type:pr state:open", se.SHA, org, repo)))
 	if err != nil {
```
Comment (foxish) [inline prow/plugins/cla/cla.go]: Log here for the number of PRs found?

File: prow/plugins/cla/cla.go (hunk 1 above)
Comment (spxtr) [inline prow/plugins/cla/cla.go]: We already have repo, I added org in the next commit.

File: prow/plugins/cla/cla.go (hunk 2 above)
Comment (foxish) [inline prow/plugins/cla/cla.go]: I would say we should log the PR#s found as well, if it has more than 1. What do you think @spxtr?

Comment (spxtr) [inline prow/plugins/cla/cla.go]: Done.

**Comment by spxtr** [comment]
LGTM

**Pull request merged**